docs/sources/administration/view-server-stats.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "View server stats"
+type = "docs"
+keywords = ["grafana", "server", "statistics"]
+[menu.docs]
+weight = 1150
++++
+
+# View Grafana server stats
+
+If you are a Grafana server admin, then you can view useful statistics about your Grafana server in the Stats tab.
+
+> **Note:** Only Grafana server admins can access the **Server Admin** menu. For more information about about admin permissions, refer to [Grafana server admin]({{< relref "../permissions/_index.md" >}}).
+
+## View server stats
+
+1. Log in to your Grafana server with an account that has the Grafana Admin flag set.
```

Comment:
I would suggest changing to something like: `Log in to your Grafana server with an account with administrator permissions`.
See my respective comment for the server settings.

Reply:
Same as above. We have two administrator types (three if you're talking about Cloud), so we have to specify which admin type we mean here.

Reply:
Correction: We have four administrator types if you count Cloud.